data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,281 @@ mission "Arachi Cartoonist 2"
 				accept
 			label noship
 			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`
+
+
+
+mission "Coalition Games"
+	name "Coalition Athletes"
+	description "Transport three athletes to <destination> by <date> where they will compete in the Coalition games."
+	deadline
+	passengers 3
+	source
+		near "Homeward" 3 12
+		government "Coalition"
+	destination "Far Home"
+	to offer
+		"coalition jobs" >= 35
+		month == 7
+	on offer
+		require "Jump Drive"
+		conversation
+			`You're approached by a group consisting of a Saryd, Kimek, and Arach. The Saryd speaks, and the small box around his neck translates after he stops.`
+			`	"The astonishing sojourner, with great joy we salute."`
+			`	You greet the group, and the Kimek continues the conversation. "The season of joy and camaraderie, we find ourselves in. Upon us, the Coalition games are. To <destination> by <day>, we must go, for competing in the games, we are. Transport us, would you? Pay you <payment>, we would."`
+			choice
+				`	"That sounds good. I'll bring you there."`
+					goto end
+				`	"Coalition games?"`
+				`	"Sorry, I'm not looking for any transport jobs right now."`
+					decline
+			`	"To determine the athlete with the most merit, a competition it is," the Saryd says.`
+			`	"A candidate for the rank of Heliarch, one becomes, if among the victors, othey are. Trained hard, we have, and to prove our worth in the games, we wish," the Arach adds, and they once again look at you, expecting an answer to their original question.`
+			choice
+				`	"Sounds like you're all really excited about this. Alright, I'll bring you there."`
+				`	"Well, I wish you luck then, but sadly I'm not looking for any transport jobs right now."`
+					decline
+			label end
+			`	"Most grateful, we are," the Saryd says.`
+			`	They introduce themselves as you help bring their luggage into your ship; the Saryd is called Ulmodi, the Kimek is Pairri, and the Arach is Tumub. They settle in their bunks to rest for the upcoming event, and you plot out a route to <destination>.`
+				accept
+	on complete
+		"coalition jobs" += 3
+		payment 300000
+		conversation
+			`While Ulmodi begins unloading the athletes' luggage and Tumub counts out your payment, Pairri approaches you. She stretches her legs trying to raise herself up your height, and you humor her by crouching down. In response, she offers you a small coin that you don't recognize, then says, "Incapable of arriving on time, a mutual friend of ours was. Available, their ticket is."  The coin has a symbol similar to the old Olympic Games on Earth, except with three interlocked rings instead of five.`
```

Comment:
```suggestion
			`While Ulmodi begins unloading the athletes' luggage and Tumub counts out your payment, Pairri approaches you. She stretches her legs trying to raise herself up to your height, and you humor her by crouching down. In response, she offers you a small coin that you don't recognize, then says, "Incapable of arriving on time, a mutual friend of ours was. Available, their ticket is."  The coin has a symbol similar to the old Olympic Games on Earth, except with three interlocked rings instead of five.`
```